gatsby_fp/src/components/Layout/Body/Fields/GenInfoTable.js

```diff
@@ -25,12 +26,23 @@ export default function GenInfoTable({
         </div>
       )} 
       <div className="flex_box">
-        {imgsrc && (
+        {fluidData && (
           <div className="img_wrapper">
-            <img
-              src={`data:image/jpeg;base64,${imgsrc}`}
-              className="bioimage"
```

Comment:
If we're no longer going to be using this class name, we should also remove any CSS that selects using this class.